manifests/c/Chocolatey/Chocolatey/2.2.2.0/Chocolatey.Chocolatey.locale.en-US.yaml

```
@@ -12,5 +12,6 @@ PackageUrl: https://github.com/chocolatey/choco
 License: Apache v2
 LicenseUrl: https://github.com/chocolatey/choco/blob/develop/LICENSE
 ShortDescription: Chocolatey is The Package Manager for Windows. It was designed to be a decentralized framework for quickly installing applications and tools that you need. It is built on the NuGet infrastructure and uses PowerShell as its focus for delivering packages.
+InstallationNotes: The Chocolatey CLI MSI is intended for installation only! If you are upgrading from 5.x of Chocolatey Licensed Extension, or  1.x of other Chocolatey products, see the upgrade guide at https://docs.chocolatey.org/en-us/guides/upgrading-to-chocolatey-v2-v6 before continuing. Otherwise, run choco upgrade chocolatey

```

```suggestion
InstallationNotes: The Chocolatey CLI MSI is for installation only! If you are upgrading from 5.x of Chocolatey Licensed Extension, or 1.x of other Chocolatey products, see the upgrade guide at https://ch0.co/v2 before continuing. Otherwise, run choco upgrade chocolatey.
```

@pauby when I did the initial validation, it was with a shorter installation note. If my counting in vscode is accurate, this message is 252 characters, but would require the short url being created.

---

Leave that with me @corbob and I'll get it sorted.

---

Sounds good. I think I have invited you to contribute on my fork of the repo so you can update this PR directly. If that doesn't work, then I guess we can always open a new PR 🤷